grouploader.py

```diff
@@ -365,6 +365,11 @@ def addCharacter(self, char, results, classes):
                 if slot in items:
                    avgilvleq = avgilvleq + items[slot]['itemLevel']
                    numitems = numitems + 1
+            # if there's no offhand, assume the main hand is a 2-hander and count it double per Blizzard iLvl formula.
+            if not 'offHand' in items:
+                avgilvleq = avgilvleq + items['mainHand']['itemLevel']
```

Comment:
This will break if neither weapon was equipped, because `mainHand` won't be there either. Otherwise looks fine.